packages/web-react/src/components/Field/useAriaIds.tsx

```diff
@@ -0,0 +1,29 @@
+import { useCallback, useState } from 'react';
+
+export type RegisterParams = { add?: string; remove?: string };
+export type RegisterType = (params: RegisterParams) => void;
+export type UseAriaIdsHook = (otherAriaIds?: string) => [string[], RegisterType];
+
+const useAriaIds: UseAriaIdsHook = (otherAriaIds) => {
+  const [ids, setIds] = useState<string[]>(otherAriaIds ? otherAriaIds.split(' ') : []);
+
+  const register = useCallback(function registerFn({ add, remove }: RegisterParams) {
```

Comment:
Is a named function required here? Can we just use the anonymous arrow function instead?

Reply:
yes i can use arrow function, good point